src/ApiPort/ConsoleProgressReporter.cs

```diff
@@ -89,17 +89,28 @@ private static void WriteColorLine(string message, ConsoleColor color)
 
         public void Dispose()
         {
-            if (!_disposed)
+            Dispose(true);
```

Comment:
Another option here would be to seal the type. Is that possible?

Reply:
Yes. Sealing the type would be a lot easier. I'll do that for both `ProgressReporters`